### ISSUE TITLE
Replace coated circuit boards with frames to make primitive circuits more distinct

### ISSUE THREAD

**Issue by doniascion** [m2r1k5/nomi-ceu#1118]
Related to #1116 

Refined circuits are not very distinct from primitive circuits. Refined is basically just a road bump on the way to microprocessors. Furthermore using wood smeared with tree goop for electronics is just a bit gross.

To improve this, change the identity of primitive circuits for a clearer tech progression.

### Primitive circuits
The first electronic components, but not actual boarded components. Assembled by hand, they are just a mess of components wired together inside a chassi. 

https://en.wikipedia.org/wiki/Point-to-point_construction

![Motorolagoldenviewchassis](https://github.com/user-attachments/assets/c9739594-9ffe-4e79-8c43-2d4f55174b1e)
![Point_to_point_wiring](https://github.com/user-attachments/assets/46d2d637-76a1-4d68-9039-dc978a19160e)

This is closer how early electric circuits were actually integrated into devices. We didn't have chips and boards with circuit paths, we had a bunch of components just wired together and attached to something like a chassis, represented by a steel frame.

Foil would not be used in primitive circuits in any way for the conductor between the components, instead we just use the cables that are already in the recipe. These are covered by the cable already in the recipe. We could possibly increase the number of cables if needed.

### Refined circuits
These would be the first boarded circuits, using phenolic resin paper boards. This is detailed in #1116 .

### Actual changes proposed

1. Remove coated boards and replace the steel plate in the primitive circuit recipe with a steel frame.
2. Increase the number of cables needed
3. Changes in #1116.

### Simpler alternative
1. Replace the steel plate with a frame with no other major changes

### Balance effects

1. Removes Coated Boards as an early gate
2. Increases overall steel cost
3. Greatly reduces copper cost (no copper wires for the boards) for less grind in the early game

**Comment by tim-burget** [comment]
> Furthermore using wood smeared with tree goop for electronics is just a bit gross.

I don't think this is a valid reason for changing the recipe. The actual process is abstracted away.

> The first electronic components, but not actual boarded components. Assembled by hand, they are just a mess of components wired together inside a chassi.

> https://en.wikipedia.org/wiki/Point-to-point_construction

You might want to read that Wikipedia article again. I think you missed something:

> Before point-to-point connection, electrical assemblies used screws or wire nuts to hold wires to an insulating wooden or ceramic board.... The crucial invention was to apply soldering to electrical assembly.

That means the wooden circuit boards that are already in the modpack are much closer to the actual first circuits, and the method you're suggesting used solder, which isn't available until much later.

Also:

> Point-to-point wiring is not suitable for automated assembly

Bit odd of a choice for a modpack about *automation*.

> We didn't have chips and boards with circuit paths, we had a bunch of components just wired together and attached to something like a chassis, represented by a steel frame.

First, what would you call the red thing in the center if not a board? Other than the different material used, Coated Circuit Boards seem like a pretty close approximation to that. Second, LV machines already include a steel chassis as is. It's called an LV Machine Hull. This idea would include a separate steel frame for every circuit a machine uses, and most machines use multiple. Plus, a number of the things that use basic circuits (like the LV Robot Arm) don't even *have* that sort of steel frame chassis. This makes no sense.

> 2. Increases overall steel cost
> 3. Greatly reduces copper cost (no copper wires for the boards) for less grind in the early game

This only "greatly reduces copper cost" before you have the wiremill. Once you have that, it only saves one copper ingot per circuit.

Let's look at the cost changes per circuit for various combinations of machines:
* No machines: +2 steel, -8 copper
* Wiremill only: +2 steel, -1 copper
* Metal Bender only: +3 steel, -2 copper
* Lathe only: -8 copper (both versions cost the same amount of steel)
* Wiremill and Metal Bender only: +3 steel, -1 copper
* Wiremill and Lathe: -1 copper
* Metal Bender and Lathe: +1 steel, -8 copper
* All three machines: +1 steel, -1 copper

Note that your suggestion actually *increases* the amount of resources needed for making circuits compared to what's currently in the pack if you for the machines that would otherwise give the most bang for your buck, the wiremill and the metal bender. The metal bender in particular saves a whopping *nine* steel ingots per machine.

In addition, copper is way quicker to obtain than steel. Lets look at how long it takes to process copper vs steel from raw ore with an unupgraded furnace and no alloy smelter:
* Copper ingots take one smelting operation per ingot, so two operations, or twenty seconds for two ingots 10 seconds/operation.
* Two steel ingots take two smelting operations to get two iron ingots, and then another two smelting operations to get two wrought iron ingots. One more smelting operation for two coal gives a total of five furnace operations, or 50 seconds. Add in the 20 seconds per ingot to actually make the steel, and that's a total of 90 seconds for two steel ingots.

Total processing time would be 80 seconds for eight copper and 90 seconds for two steel, which *might* be short enough for the extra ore needed for the copper to make a different, but that amount of ore really doesn't take long to obtain.

Finally, once you have your basic machines, it takes longer (by almost five seconds) to produce a steel frame from steel than it does to produce the eight fine copper wires used in the current craft.



In summary: The steel chassis in your suggestion is already represented by the LV Machine Hull and thus doesn't need to be represented by a steel frame, your suppose decrease in grinding would be negligible before obtaining the wiremill or bending machine and would actually result in an *increase* in grinding afterward, and your supposedly "first electronic components" actually came *after* the circuits already represented as the first circuits in the pack in real life, anyway.
